frontend/components/atoms/images/main-logo.php

```diff
@@ -18,7 +18,7 @@
 		);
 		?>
 	<?php else : ?>
-		<div class="inline font-bold uppercase text-small">
+		<div class="inline font-bold uppercase text-1.4">
```

Comment:
I think this 1.4 is too specific, maybe we should keep it as text-small and then modify it based on each project requirements.

Reply:
I'll update